### PR TITLE
{bio}[*/2015a] EMAN 2.11 + LibTIFF 4.0.4 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires #1733 and ~~#1738~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ae32947f01de252bf27e for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 18 out of 21 (4 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2932ea1621bb941f6a49 for a full test report.

**Comment by boegel** [comment]
seems like FTGL (http://sourceforge.net/projects/ftgl/) is missing as a dependency in the EMAN easyconfigs...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 5 (4 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7e161e28e200ce9056f3 for a full test report.

File: easybuild/easyconfigs/e/EMAN2/EMAN2-2.11-foss-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
+ with a primary focus on processing data from transmission electron microscopes. """
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://ncmi.bcm.edu/ncmi/software/counter_222/software_130/']
+sources = ['eman%(version)s.source.tar.gz']
+
+pyver = '2.7.9'
```
Comment (wpoely86) [inline easybuild/easyconfigs/e/EMAN2/EMAN2-2.11-foss-2015a-Python-2.7.9.eb]: .10 ?

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/e/EMAN2/EMAN2-2.11-foss-2015a-Python-2.7.9.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/e/EMAN2/EMAN2-2.11-foss-2015a-Python-2.7.9.eb]: I need it in a .9 context, so I'll stick to .9 for now, bumping to .10 is trivial/left to the interested reader

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1ae841214b556f5dd835 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**